content/en/docs/contribute/start.md

```diff
@@ -294,15 +294,48 @@ contribution guide.
 
 ## Review docs pull requests
 
-People who are not yet approvers or reviewers can still review pull requests. 
-The reviews are not considered "binding", which means that your review alone 
-won't cause a pull request to be merged. However, it can still be helpful. Even 
-if you don't leave any review comments, you can get a sense of pull request 
-conventions and etiquette and get used to the workflow. Take a look at the 
-[Content](/docs/contribute/style/content-guide/) and 
-[Style](/docs/contribute/style/style-guide/) guides before you review so you 
+People who are not yet approvers or reviewers can still review pull requests. You
+learn the code base, and you begin to build trust with your fellow contributors.
+You don't need to be a good developer or writer to review pull requests, but you do need
+to be a good human being. This is a community that uses English for written and
+verbal communication. Contributor's English skills are at different levels, so
+patience and clear, concise feedback are very important for maintaining good
+community relationships!  If you show yourself as thoughtful, polite, and
+diligent through your reviews, other reviewers will be more likely to prioritize
+your pull requests and collaborate on your ideas. To borrow from Mark McLoughlin's [An
+Ideal OpenStack Developer](https://blogs.gnome.org/markmc/2014/06/06/an-ideal-openstack-developer/)
+post, you need to know when to be
+[empathetic](https://www.merriam-webster.com/dictionary/empathetic), when to be
+[pragmatic](https://www.merriam-webster.com/dictionary/pragmatic), and when to
+be [dogmatic](https://www.merriam-webster.com/dictionary/dogmatic). A good
+reviewer focuses both on the big picture and the little details.
+
+The reviews are not considered "binding", which means that your review alone
+won't cause a pull request to be merged. However, it can still be helpful. Even
+if you don't leave any review comments, you can get a sense of pull request
+conventions and etiquette and get used to the workflow. Take a look at the
+[Content](/docs/contribute/style/content-guide/) and
+[Style](/docs/contribute/style/style-guide/) guides before you review so you
 get an idea of what the content should contain and how it should look.
 
+### Best practices
+
+- Be polite, considerate, and helpful; it's your job to help people get PRs merged
+- If you are asking for a lot of changes, make sure at least one comment is positive
+- Comments should be constructive rather than destructive; before leaving a
+  comment, minimize conflict by thinking about how defensive the PR's author may
+  feel when reading your comment
+- Give the PR author the benefit of the doubt; for example, if the author
```

Comment:
```suggestion
- Assume good intent and ask clarifying questions
```

Reply:
I admit you got me on "the benefit of the doubt" as an idiom